app/controllers/api/auth/episodes_controller.rb

```diff
@@ -26,6 +26,6 @@ def visible?
   end
 
   def resources_base
-    @episodes ||= authorization.token_auth_episodes.merge(super)
+    @episodes ||= super.merge(authorization.token_auth_episodes)
```

Comment:
Had to reverse the order of this merge, so the `token_auth_episodes` (which contains the `with_deleted`) can override the `super` query, which does not include deleted.